Makefile

```diff
@@ -74,9 +74,9 @@ available-node = \
 # BUILDTYPE=Debug builds both release and debug builds. If you want to compile
 # just the debug build, run `make -C out BUILDTYPE=Debug` instead.
 ifeq ($(BUILDTYPE),Release)
-all: out/Makefile $(NODE_EXE) ## Default target, builds node in out/Release/node.
+all: $(NODE_EXE) ## Default target, builds node in out/Release/node.
 else
-all: out/Makefile $(NODE_EXE) $(NODE_G_EXE)
```

Comment:
I’m not sure I understand why these have become unnecessary?

Reply:
They are direct dependent of these targets:
https://github.com/nodejs/node/blob/f0725e5238bd50b97d55e758effb7579e871cc87/Makefile#L98-L102

Reply:
Okay, that makes sense!